scripts/build.ps1

```diff
@@ -118,10 +119,10 @@ if (!$PSVersionTable.Platform -or $IsWindows) {
         Copy-Item "$buildoutput\v8jsi.dll.pdb" -Destination "$OutputPath\lib\$AppPlatform\$Configuration\$Platform"
     }
 
-    # Debugging extension
-    if ($AppPlatform -ne "uwp") {
-        Copy-Item "$buildoutput\v8windbg.dll" -Destination "$OutputPath\lib\$AppPlatform\$Configuration\$Platform"
-    }
+    # Skip Debugging extension

```

Comment:
>Skip Debugging extension [](http://example.com/codeflow?start=6&length=24)

We do want to publish this into the NuGet as it can be manually retrieved for use in debugging (it needs to version match v8jsi.dll). Is there a way to skip scanning it instead of removing it altogether?

Reply:
From the docs there is no way to exclude it from binskim...
@tudorms : The only option to get it passing is to rename run tool and name back.. But that is agsinst the spirit of compliance. I would not recommend it... We should either get it compliant or not ship it...

Reply:
Resolved with Tudor